.github/workflows/mvn-verify-check.yml

```diff
@@ -42,10 +42,8 @@ jobs:
         id: noSnapshotVersionsStep
         run: |
           set -x
-          noSnapshotVersionsStr=$(mvn -B help:evaluate -q -pl dist -PnoSnapshots -Dexpression=included_buildvers -DforceStdout)
-          noSnapshotVersionsStr=$(echo $noSnapshotVersionsStr)
-          noSnapshotVersionsArr=($(IFS=", "; echo $noSnapshotVersionsStr))
```

Comment:
need to pass something to produce headVersion. this is referenced by L50 `echo ::set-output name=headVersion::${noSnapshotVersionsArr[0]}` and thus the workflow is failing

Reply:
after refactoring common.sh now is used only in version-def.sh, so we could just inline its contents there and drop common.sh altogether :)